src/llama_stack/telemetry/constants.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
any particular reason you want to hoist this to top-level vs. the folder being in `core/` subdirectory?

Reply:
I can move it if you would prefer it. I viewed it as its own package, and wanted it to be easy to import so I put it there, but it can go anywhere.